src/commands/ping.ts

```diff
@@ -0,0 +1,95 @@
+import chalk from 'chalk'
+import * as arg from 'arg'
+
+import { ping, LANDPingResponse } from '../web3/ping'
+import { Decentraland } from '../lib/Decentraland'
+import { Analytics } from '../utils/analytics'
+import { isValid, getObject, Coords } from '../utils/coordinateHelpers'
+import { fail, ErrorType } from '../utils/errors'
+import { parseTarget } from '../utils/land'
+import { formatDate } from '../utils/dates'
+
+export const help = () => `
+  Usage: ${chalk.bold('dcl ping [target] [options]')}
+
+    ${chalk.dim('Options:')}
+
+      -h, --help                Displays complete help
+      -n, --network             Choose between ${chalk.bold('mainnet')} and ${chalk.bold(
+  'ropsten'
+)} (default 'mainnet')
+
+
+    ${chalk.dim('Examples:')}
+
+    - Get LAND liveness state of the current project"
+
+      ${chalk.green('$ dcl ping')}
+
+    - Get LAND liveness state of ${chalk.bold('-12, 40')}"
+
+      ${chalk.green('$ dcl ping -12, 40')}
+`
+
+function getStatus(pingResponse: LANDPingResponse): string {
+  if (pingResponse.hasDecayed) {
+    return chalk.red('Decayed')
+  }
+
+  if (pingResponse.isNearDecay) {
+    return chalk.yellow('Near to decay')
```

Comment:
```suggestion
    return chalk.yellow('Near decay')
```

is proper*er* english